tests/Users/Controllers/UserControllerTest.php

```diff
@@ -270,6 +270,19 @@ public function testBrowseFound()
         }
     }
 
+    public function testBrowseWithOrder()
+    {
+        $user = factory(App\User::class)->create();
+        $user->name = 'B';
+        $user->save();
+
+        $res = $this->call('GET', '/users?sort=name&direction=desc');
+        $this->assertEquals(200, $res->getStatusCode());
+        $results = json_decode($res->getContent());
+        $this->assertEquals('B', $results->entities[0]->name);
+        $this->assertEquals('Administrator', $results->entities[1]->name);
+    }
+
```

Comment:
Chỗ test em nên viết thêm vài case nữa. Ví dụ như người ta không truyền vào sort, không truyền vào direction. Cũng nên tạo thêm nhiều item hơn bằng cách sử dụng for